specification/2.0/appendix/performance-metrics/README.md

```diff
@@ -0,0 +1,31 @@
+
+--- Name -----------Property    ----- Description ----------------------------
+[verticecount]      Scene       Total number of vertices used by a model in a scene
```

Comment:
It's `vertexcount`, not `verticecount`.  The singular form of "vertices" is "vertex."